src/System.Private.Uri/tests/FunctionalTests/UriParserTest.cs

```diff
@@ -0,0 +1,556 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// See the LICENSE file in the project root for more information.
+// Author:
```

Comment:
Do we need to keep the "author" information, or just the copyright section?  Seems like the author information is logically a separate comment that just happens to be at the top.